grpc-gateway/service/deleteResource_test.go

```diff
@@ -68,7 +67,7 @@ func TestRequestHandler_DeleteResource(t *testing.T) {
 		},
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), TEST_TIMEOUT)
+	ctx, cancel := context.WithTimeout(context.Background(), testCfg.TEST_TIMEOUT)
```

Comment:
DELETE is also missing success test path

Reply:
https://gitlab.iotivity.org/iotivity/iotivity-lite/-/merge_requests/185